sections/futures/Trade/TradeBalance.tsx

```diff
@@ -48,59 +54,116 @@ const TradeBalance: React.FC<TradeBalanceProps> = memo(({ isMobile = false }) =>
 
 	return (
 		<Container>
-			<FlexDivRowCentered>
-				<BalanceContainer clickable={accountType === 'cross_margin'} onClick={onClickContainer}>
-					{accountType === 'cross_margin' && isDepositRequired ? (
-						<FlexDivRowCentered>
-							<FlexDivCol>
+			<BalanceContainer clickable={accountType === 'cross_margin'} onClick={onClickContainer}>
+				{accountType === 'cross_margin' && isDepositRequired ? (
+					<DepositContainer>
+						<FlexDivCol>
+							<FlexDivRow columnGap="5px" justifyContent="flex-start">
 								<Body size={isMobile ? 'small' : 'medium'} color="secondary">
 									No available margin
-									{expanded ? <HideIcon /> : <ExpandIcon />}
-								</Body>
-								<Body size={isMobile ? 'small' : 'medium'} color="preview">
-									Min. $50 sUSD required to trade
 								</Body>
-							</FlexDivCol>
-							<Button
-								variant="yellow"
-								size="xsmall"
-								textTransform="none"
-								onClick={() => dispatch(setOpenModal('futures_isolated_transfer'))}
-							>
-								{t('header.balance.get-susd')}
-							</Button>
-						</FlexDivRowCentered>
-					) : (
-						<FlexDivCol>
-							<Body size={isMobile ? 'medium' : 'large'} color="secondary">
-								Available Margin
-								{accountType === 'cross_margin' ? expanded ? <HideIcon /> : <ExpandIcon /> : null}
+								{expanded ? <HideIcon /> : <ExpandIcon />}
+							</FlexDivRow>
+							<Body size={isMobile ? 'small' : 'medium'} color="preview">
+								Min. $50 sUSD required to trade
 							</Body>
-							<NumericValue size={isMobile ? 'medium' : 'large'} weight="bold">
-								{accountType === 'isolated_margin'
-									? formatDollars(availableIsolatedMargin)
-									: formatDollars(idleMargin)}
-							</NumericValue>
 						</FlexDivCol>
-					)}
-				</BalanceContainer>
+						<Button
+							variant="yellow"
+							size="xsmall"
+							textTransform="none"
+							onClick={() => dispatch(setOpenModal('futures_isolated_transfer'))}
+						>
+							{t('header.balance.get-susd')}
+						</Button>
+					</DepositContainer>
+				) : (
+					<>
+						{isMobile ? (
+							<FlexDivCol style={{ rowGap: '2px' }}>
```

Comment:
There's a lot of duplicated code here just for col v row, I think it would be better to create styled components and pass `isMobile` to for determining the flex.

Reply:
Yes. Good idea. I will do that after merging #2361